gdal/fuzzers/build_google_oss_fuzzers.sh

```diff
@@ -84,6 +84,7 @@ build_ogr_specialized_fuzzer bna RegisterOGRBNA "/vsimem/test.bna" "/vsimem/test
 build_ogr_specialized_fuzzer wasp RegisterOGRWAsP "/vsimem/test.map" "/vsimem/test.map"
 build_ogr_specialized_fuzzer xlsx RegisterOGRXLSX "/vsimem/test.xlsx" "/vsitar/{/vsimem/test.xlsx}"
 build_ogr_specialized_fuzzer ods RegisterOGRODS "/vsimem/test.ods" "/vsitar/{/vsimem/test.ods}"
+build_ogr_specialized_fuzzer lvbag RegisterOGRLVBAG "/vsimem/test.ods" "/vsitar/{/vsimem/test.xml}"
```

Comment:
As a lvbag dataset only comprises of a single xml file, the following should be used:
```suggestion
build_ogr_specialized_fuzzer lvbag RegisterOGRLVBAG "/vsimem/test.xml" "/vsimem/test.xml"
```
what you copied was for drivers where a dataset is a set of files, and so they must be zipped/tared